src/cone_finder/scripts/cone_finder_fake.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+import rospy
+import math
+from amp_msgs.msg import ConeList
+
+
+def fake_loop():
+    rate = rospy.Rate(rospy.get_param("cone_finder/update_rate"))
+    pub = rospy.Publisher("cones_found", ConeList, queue_size=10)
+    while not rospy.is_shutdown():
+        pub.publish(fake_cones)
+        rate.sleep()
+
+
+if __name__ == "__main__":
+    # Generates two concentric rings of cones, clumped in groups of 3
+    inner_radius = 1
+    inner_count = 8
+    outer_multiplier = 3
+
+    """fake_cones = ConeList()
+    fake_cones.x = []
+    fake_cones.y = []
+    for i in range(int(inner_count * outer_multiplier)):
+        rad = (i / (inner_count * outer_multiplier)) * math.pi * 2 - (i % 3) * 0.15
+        # outer ring
+        fake_cones.x.append(math.cos(rad) * inner_radius * outer_multiplier)
+        fake_cones.y.append(math.sin(rad) * inner_radius * outer_multiplier)
+
+        # inner ring, less dense
+        if i % outer_multiplier == 0:
+            fake_cones.x.append(math.cos(rad) * inner_radius)
+            fake_cones.y.append(math.sin(rad) * inner_radius)"""
```

Comment:
Is this something that needs to be cleaned up?

Reply:
Left it in for testing, incase we modify the algorithm in the future. The "fake" node won't be used much anyway, so I'm not concerned leaving commented-out code in it.

Reply:
I see, could you add it as an option that can be toggled with a flag from an environment environment or just an `if false` branch?